autode/wrappers/ORCA.py

```diff
@@ -190,6 +190,34 @@ def print_coordinates(inp_file, molecule):
     return
 
 
+def calc_atom_entropy(atom_label, temp):
+    """
+    Calculate the entropy of a single atom, not available in ORCA as only the
+    translational contribution: S_trans = R (ln(q_trans) + 5R/2)
+
+    Arguments:
+        atom_label (str):
+        temp (float): Temperature in K
+
+    Returns:
+        (float):
+    """
+
+    k_b = 1.38064852E-23          # J K-1
```

Comment:
should these be in autode.constants.py?

Reply:
Probably, but autode.constants.Constants is currently just a bunch of conversion factors.. probably not a major problem(?)